### DISCUSSION TITLE
dynamic annotations

### DISCUSSION THREAD

**Discussion by fdouglis** [coreemu/core#556] [Q&A]
Let's say I have a script that is connecting nodes, moving them around, and so on, and I want to draw attention to what is happening at any given point.

I note that the GUI has an "annotations" panel that will allow me to manually draw a circle, and put a label in it -- which is exactly the sort of thing I want to add.  But that panel only appears when the emulation is stopped.  And in addition, I really want to control it programmatically.  I've looked at the `coresendmsg` script/libraries and documentation and I don't see a way to send it something saying to add or remove such an annotation?  

Is it possible without changing the server?

**Comment by bharnden** [comment]
Sorry, not currently, it is GUI specific functionality. All current APIs interface with the `core-daemon` which knows nothing of annotations.

Something to consider for the future though.

**Reply by fdouglis** [reply]
Thanks, @bharnden, for the prompt reply.  I was afraid of that.
